lib/matplotlib/quiver.py

```diff
@@ -240,6 +243,8 @@ def __init__(self, Q, X, Y, U, label, **kw):
         self.color = kw.pop('color', None)
         self.label = label
         self._labelsep_inches = kw.pop('labelsep', 0.1)
+        self._vswap = kw.pop('vswap', False)
```

Comment:
And where you are at it can you move the rest of the things that use `pop` up into the signature?